### ISSUE TITLE
Double Dragon Neon (252350)

### ISSUE THREAD

**Issue by ryanmusante** [ValveSoftware/Proton#2128]
# Whitelist Request
- Name of the game to be whitelisted: Double Dragon Neon
- Steam AppID of the game: 252350

## System Information
- GPU: Geforce GTX 980M
- Driver/LLVM version: nvidia 415.22.01
- Distro version: Manjaro Linux 
- Link to full system information report as [Gist](https://gist.github.com/):https://gist.github.com/ryanmusante/e0789f11ca3adfa4442297ac0099461f
- Proton version: 3.16-6 beta

## I confirm:
- [x ] that pressing the `Play` button in the Steam client is sufficient.
- [ ] that [runtime config options](https://github.com/ValveSoftware/Proton#runtime-config-options) are necessary to run the game.
- [x ] that no workarounds other than PROTON_* env variables are necessary.

## Issues
- [x ] I haven't experienced any issues.
- [x ] There are no issues left open for this game.
- [x ] Although I consider the gaming experience equal to Windows there are remaining issues:

**Comment by opal1974** [comment]
Someone on protondb is reporting that this game crashed for him when  he selected starting new game, though I note he was using Mesa drivers.

**Comment by AwesamLinux** [comment]
I have tried this game with every version of Proton, and I have never been able to play more than a couple of minutes before it crashes. It almost always crashes when selecting to start new game at the main menu. Here is a log of when exactly that happens.

[steam-252350.log](https://github.com/ValveSoftware/Proton/files/3108394/steam-252350.log)
[system_info.txt](https://github.com/ValveSoftware/Proton/files/3108395/system_info.txt)

Distro: Linux Mint 19.1 Tessa (64 bit)
GPU: Vega 56
Driver/LLVM version: Mesa 19.1.0-devel/9.0.0
Kernel version: 5.0.9-050009-generic
Proton version: 4.2-3

**Comment by opal1974** [comment]
Yeah, it seems that it has something to do with Mesa drivers.  Nvidia it is perfect.

**Comment by AwesamLinux** [comment]
## System Information
- GPU: RX VEGA 56
- GPU Driver: Mesa ACO 19.1.0+aco+git1564433637-5c6077221dd-1bionic2 (Valves PPA)
- Kernel version:  5.2.5-050205-lowlatency
- Distro version: Linux Mint 19.1 Tessa (64 bit) (Cinnamon Desktop)
- Proton version: 4.11-1
- System information: [system_info.txt](https://github.com/ValveSoftware/Proton/files/3458172/system_info.txt)

## Compatibility Report
As of Proton 4.11-1 game works perfectly out of the box, no crashes. 

- Good performance, solid 60fps at all times
- Nintendo Switch Pro controller worked
- Windowed, Fullscreen and Borderless mode works
- Switching between desktop and the game works 

With Proton 4.2-9 and 3.16-9 and the game still crashes. So I think actually the crashing issue was Proton related and not the Mesa drivers.

I also tested with **D9VK on Proton 4.11-1 and with that the game crashes directly on launch**. This game is as far as I know is a DX9 game, so maybe there is an issue with D9VK. 

Logs of all of these tests:
[steam-252350-D9VK-fail.log](https://github.com/ValveSoftware/Proton/files/3458175/steam-252350-D9VK-fail.log)
[steam-252350-proton-3.16-9-fail.log](https://github.com/ValveSoftware/Proton/files/3458176/steam-252350-proton-3.16-9-fail.log)
[steam-252350-proton-4.2-9-fail.log](https://github.com/ValveSoftware/Proton/files/3458177/steam-252350-proton-4.2-9-fail.log)
[steam-252350-proton-4.11-1b-success.log](https://github.com/ValveSoftware/Proton/files/3458178/steam-252350-proton-4.11-1b-success.log)

**Comment by AwesamLinux** [comment]
**System Information**
- GPU: RX VEGA 56
- GPU Driver: 19.1.0+aco+git1566739320-db73bde35cc-1bionic1 (Valves PPA)
- Kernel version: 5.2.10-050210-lowlatency
- Distro version: Linux Mint 19.2 Tina (64 bit) (Cinnamon)
- Proton version: 4.11-3

**Retest Compatibility Report**

Game still works perfectly on Proton 4.11-3 out of the box. 

With D9VK the game now crashes when starting a new game (both with ACO and LLVM) from the games main menu. So the **difference with 4.11-3** is that with D9VK the game no longer crashes directly on launch as it did with 4.11-1.

[steam-252350-proton-4.11-3.log](https://github.com/ValveSoftware/Proton/files/3561833/steam-252350-proton-4.11-3.log)
[steam-252350-proton-4.11-3-d9vk-aco.log](https://github.com/ValveSoftware/Proton/files/3561834/steam-252350-proton-4.11-3-d9vk-aco.log)
[steam-252350-proton-4.11-3-d9vk-llvm.log](https://github.com/ValveSoftware/Proton/files/3561835/steam-252350-proton-4.11-3-d9vk-llvm.log)
[system_info.txt](https://github.com/ValveSoftware/Proton/files/3561836/system_info.txt)

**Comment by jonlap** [comment]
# Whitelist Request

## System Information
- GPU: RX 480
- Driver/LLVM version: Mesa 19.3.1/9.0.0 (ACO enabled)
- Distro version: Arch Linux
- Link to full system information report as [Gist](https://gist.github.com/): My [Steam info](https://gist.github.com/jonlap/9fd1c7791ed8db6d2e298084418ee726)
- Proton version: 4.11-11

## I confirm:
- [x] that pressing the `Play` button in the Steam client is sufficient.
- [ ] that [runtime config options](https://github.com/ValveSoftware/Proton#runtime-config-options) are necessary to run the game.
- [x] that no workarounds other than PROTON_* env variables are necessary.

## Issues
- [x] I haven't experienced any issues.
- [x] There are no issues left open for this game.
- [ ] Although I consider the gaming experience equal to Windows there are remaining issues:


## Notes
Did a playthrough of the game. Everything works as expected.

**Comment by kisak-valve** [comment]
[252350] Double Dragon Neon: crashes at launch

Issue transferred from https://github.com/ValveSoftware/Proton/issues/3707.
@toalex77 posted on 2020-03-30T18:16:57:

# Compatibility Report
- Name of the game with compatibility issues: Double Dragon Neon
- Steam AppID of the game: 252350

## System Information
- GPU: nVidia GTX 970
- Driver/LLVM version: nVidia 440.66.03
- Kernel version: 5.5.11
- Link to full system information report as [Gist](https://gist.github.com/toalex77/051a95a0cbdd8d823dadadace4f07cc1/):
- Proton version: 5.0-5

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-252350.log](https://github.com/ValveSoftware/Proton/files/4404628/steam-252350.log)



## Symptoms 

Game starts, then suddenly crashes.

## Reproduction

I suppose the issue is related to the 32 cores of my system, in fact, if I turn off some of the cores, the game starts and it's playable.

**Comment by kisak-valve** [comment]
Hello @toalex77, you might get lucky trying something like `taskset -c 0-15 %command%` in the game's launch options and checking if the game runs reliably without reducing the cpu core count in the bios.

**Comment by toalex77** [comment]
> Hello @toalex77, you might get lucky trying something like `taskset -c 0-15 %command%` in the game's launch options and checking if the game runs reliably without reducing the cpu core count in the bios.

Hi, @kisak-valve. No luck with taskset, I have to put some core offline (4 is sufficient). Normally I do this from command line with something like this: `echo 0 > /sys/devices/system/cpu/xxx/online`

**Comment by toalex77** [comment]
Hi, just an update: I've re-tested the game with latest Proton releases.
With 5.0-10 , Experimental, and also the unofficial Proton GE 5.21-GE-1) but without turning off some of the cores, the game continues to crash at launch. This time I've to turn off at least 13 cores.

**Comment by toalex77** [comment]
Another update: after a while I've tested another time this game, and even with latest Proton versions, official and unofficial, the game continues to crash at launch unless I don't turn off some cores.
But this time I've also tried to apply the customization suggested from another user on ProtonDB: https://www.protondb.com/app/252350#S2081DDsoF

Setting the Windows version to "Windows XP", solves this issue and the game now starts without I have to disable any core.

**Comment by AwesamLinux** [comment]
## System Information
- GPU: AMD Radeon RX 5700
- Driver/LLVM version: Mesa 23.0.0-git
- Kernel version: 5.18.5-201.fsync.fc36.x86_64
- Proton Version: proton-7.0-3

Game crashes on launch, setting launch option "WINE_CPU_TOPOLOGY=4:0,1,2,3 %command%" solves the problem.

(I tried taskset but that did not work, appears to be same kind of issue as mentioned in #5927)

including logs of launching the game successfully and unsuccessfully:
[steam-252350-fail.log](https://github.com/ValveSoftware/Proton/files/8968350/steam-252350-fail.log)
[steam-252350-success.log](https://github.com/ValveSoftware/Proton/files/8968351/steam-252350-success.log)
[system_info.txt](https://github.com/ValveSoftware/Proton/files/8968352/system_info.txt)

**Comment by aeschrud** [comment]
Windows XP Compatibility solved this issue for me, thanks
